packages/api/endpoints/distribution.js

```diff
@@ -78,11 +76,17 @@ function handler(event, context, cb) {
       const user = tokenInfo.endpoint.replace('/api/users/', '');
 
       // otherwise we get the temp url and provide it to the user
-      const url = s3.getSignedUrl('getObject', {
+      const signedUrl = aws.s3().getSignedUrl('getObject', {
```

Comment:
Expanation: Use the S3 client from the common module so that we get LocalStack support for testing.